File: WebApplication/src/suppliers/dtos/junctions/supplier-offering-equipment.dto.ts
import { SupplierEquipmentDto } from "../supplier-equipment.dto";
import { SupplierOfferingDto } from "../supplier-offering.dto";

export class SupplierOfferingEquipmentDto {
    id: number;

    supplierOfferingId: number;
    supplierOffering: SupplierOfferingDto;

    supplierEquipmentId: number;
    supplierEquipment: SupplierEquipmentDto;
}

File: Sc.Models/Dtos/Suppliers/Junctions/SupplierOfferingEquipmentDto.cs
using Sc.Models.Entities.Base;

namespace Sc.Models.Dtos.Suppliers.Junctions
{
    public class SupplierOfferingEquipmentDto : Entity
    {
        public int SupplierOfferingId { get; set; }
        public SupplierOfferingDto SupplierOffering { get; set; }

        public int SupplierEquipmentId { get; set; }
        public SupplierEquipmentDto SupplierEquipment { get; set; }
    }
}
